Unreal/CMakeLists.txt

```diff
@@ -55,6 +55,12 @@ set (
   ${CARLA_UNREAL_ENGINE_PATH}/Engine/Binaries/${UE_SYSTEM_NAME}/UnrealEditor${EXE_EXT}
 )
 
+if (IS_DIRECTORY ${CARLA_UE_PATH}/Content/Carla)
```

Comment:
Makes sense to move this check to the `launch` target?

Content is not needed for build, is needed just for launch. Sometimes I would like to build CARLA without launching.

Reply:
We could look into it, but the idea here is that if the variable BUILD_CARLA_UNREAL is set to ON, we expect content to be downloaded. Alternatively we could use a warning instead of an error.

Reply:
On my understanding, is good to have this error for `launch` because with out this error is difficult for the users to know why Unreal Engine is chrashing when the content is missing.

But in the other hand, users should be able to build without content.

That is why I propose move this check to the `launch` target instead of CARLA build.

Do you think it will be difficult to move this check to the `launch` target?